P1/Project_Code/frontend/src/routes/Login/Login.route.jsx

```diff
@@ -48,24 +41,50 @@ function Login() {
 		}
 	}
 
+	//Error validation function
+	const validateInput = async (data) => {
+		let inputErrors = {};
+
+		if (!isEmail(data.email))
```

Comment:
Consider adding a check that the value of data.email and data.password are not null or undefined. See example below 
```
if (!data.email || !isEmail(data.email)) {
      inputErrors.email = "Invalid Email";
}
```